### DISCUSSION TITLE
Feature Request:  Support `min` value for Progress component

### DISCUSSION THREAD

**Discussion by kartzke** [saadeghi/daisyui#2323] [Ideas / feature requests]
Hello, I have an idea that I would love to see implemented in the DaisyUI Progress component.

It is currently possible to set a `value` and a `max` value on the Progress element.  Here is the example code from the docs:

`<progress class="progress progress-warning w-56" value="40" max="100"></progress>`

Which would produce a Progress bar like this:

![image](https://github.com/saadeghi/daisyui/assets/713503/0ea4482c-a208-41d5-be6e-79093ea30613)

It would be great if we could specify an optional `min` value.   This would allow use cases for displaying more precise ranges, like historical price ranges, temperature ranges, etc.  

For example: the progress component could be used to show a product has been sold for as low as $4.42 and as high as $6.23, with a current price of $4.99.

I don't believe this is supported, and I found a similar request for the Radial progress bar: https://github.com/saadeghi/daisyui/discussions/873

**Comment by kartzke** [comment]
Another cool feature that could be added as part of this, or in the future, is control of the color.   For example, gradient colors... green to red (for price viz) or blue to red (temperature viz)... or binary colors based on progress thresholds... if the value is less than 50% of the range, make the progress bar color 1, and if it's above 50%, make it color 2.

**Comment by saadeghi** [comment]
> It would be great if we could specify an optional `min` value.

It's not my decision. The HTML `<progress>` element has [`max` and `value`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress#attributes). We can't add a non-standard `min` attribute. It should be based on HTML standards and the HTML `<progress>` element value always starts from `0`
